src/operations/cms/queries/getInternalNewsArticles.ts

```diff
@@ -20,6 +21,12 @@ export const GET_INTERNAL_NEWS_ARTICLES = gql`
         name
         type
       }
+      body {
```

Comment:
We need to return the body of the article here, since this particular query is interested in getting the entire article and not just the preview

Reply:
update: removing this, moving query into separate file